[2021-04-27 17:58:29,522] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: docker_sample.docker_op_tester 2021-04-25T05:10:00+00:00 [queued]>
[2021-04-27 17:58:29,539] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: docker_sample.docker_op_tester 2021-04-25T05:10:00+00:00 [queued]>
[2021-04-27 17:58:29,540] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-04-27 17:58:29,540] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2021-04-27 17:58:29,541] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-04-27 17:58:29,549] {taskinstance.py:1063} INFO - Executing <Task(DockerOperator): docker_op_tester> on 2021-04-25T05:10:00+00:00
[2021-04-27 17:58:29,554] {standard_task_runner.py:52} INFO - Started process 319 to run task
[2021-04-27 17:58:29,559] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'docker_sample', 'docker_op_tester', '2021-04-25T05:10:00+00:00', '--job-id', '104', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dockerOperator.py', '--cfg-path', '/tmp/tmpdp1n2x5h', '--error-file', '/tmp/tmpjp68zx6d']
[2021-04-27 17:58:29,561] {standard_task_runner.py:77} INFO - Job 104: Subtask docker_op_tester
[2021-04-27 17:58:29,608] {logging_mixin.py:104} INFO - Running <TaskInstance: docker_sample.docker_op_tester 2021-04-25T05:10:00+00:00 [running]> on host 44e91a58ae1b
[2021-04-27 17:58:29,656] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=docker_sample
AIRFLOW_CTX_TASK_ID=docker_op_tester
AIRFLOW_CTX_EXECUTION_DATE=2021-04-25T05:10:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-04-25T05:10:00+00:00
[2021-04-27 17:58:29,669] {docker.py:224} INFO - Starting docker container from image centos:latest
[2021-04-27 17:59:00,402] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=docker_sample, task_id=docker_op_tester, execution_date=20210425T051000, start_date=20210427T175829, end_date=20210427T175900
[2021-04-27 17:59:00,477] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-04-27 17:59:00,536] {local_task_job.py:146} INFO - Task exited with return code 0
